rules/msk_app_consume_groups.md

```diff
@@ -0,0 +1,41 @@
+# `msk_app_consume_groups`
+
+Requires that any element of the `consume_groups` in a `tls-app` include a `'.'`
```

Comment:
I think we should adapt this too to say that consumer groups must be prefixed with the name of the team using it since MSK is a multitenant environment and we want do know to which team a consumer group belongs.
Also, in kafka-ui, access is given to consumer groups based on the team prefixes.

Reply:
> I think we should adapt this too to say that consumer groups must be prefixed with the name of the team using it since MSK is a multitenant environment and we want do know to which team a consumer group belongs. Also, in kafka-ui, access is given to consumer groups based on the team prefixes.

[b32e67f](https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/35/commits/b32e67f0ef57fbb13755062584dd0ba75959b04d)